### ISSUE TITLE
When having multiple test components, MRO reports wrong test results

### ISSUE THREAD

**Issue by martsec** [opendevstack/ods-jenkins-shared-library#377]
**Describe the bug**
When having multiple `ods-test` components in a project, the MRO will override the test results from the components with the latter one. 

Let's say we have two test components: 

```
 - id: spock-int-repo
    type: ods-test
 - id: geb-e2e-repo
    type: ods-test
```
And you have implemented Tests 1 and 2 in spock and 3 in Geb. 
You will see the correct information in Jenkins, but not in the documents nor Jira, since the results get overwritten 

![bug-mro-test-components-jira](https://user-images.githubusercontent.com/6894585/84372688-c8024180-abdb-11ea-8e00-ca9de6fb770e.png)

**To Reproduce**
1. Have in a project (and in the release manager metadata.yml) two ods-test repositories
2. Set-up integration/acceptance/installation tests in Jira and implement them in code
3. Execute the pipeline 

**Expected behavior**
Tests results coming from multiple components should be merged before reporting to Jira and the creation of the documents.

**Affected version (please complete the following information):**
 - OpenShift: 3.11
 - OpenDevStack 2.x

**Comment by martsec** [comment]
Won't fix in ODS 3. Will be moved to 3.1 or so.
